test-data/unit/check-incremental.test

```diff
@@ -2285,7 +2289,7 @@ S = str
 import b
 def f(x: b.S) -> int: return 0
```

Comment:
The remaining tests don't have [rechecked] and [stale] fields -- not sure if this is intentional?